apps/api/src/server.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Setup express server.
+ */
+
+import express, { NextFunction, Request, Response } from 'express'
+import helmet from 'helmet'
+import logger from 'jet-logger'
+import morgan from 'morgan'

```

Comment:
Isn't morgan another logger?

Reply:
yup the generator i used makes use of two loggers, one for express stuf and another for custom logs. Opted for regular console logs and kept morgan which is a classic out-of-the-box experience for epxress apps logging